packages/app/pages/login.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components";
+import { Form, Input, Button, Checkbox } from "antd";
+import React, { useState } from "react";
+import NavBar from "../components/Nav/NavBar";
+
+const Container = styled.div`
+  width: auto;
+  height: auto;
+  @media (max-width: 768px) {
+    margin: 32px 24px;
+  }
+`;
+
+const LoginContainer = styled.div`
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+
+  padding-top: 300px;
+`;
+
+const InputBoxContainer = styled.div`
+  margin: auto;
+  width: 500px;
+  height: 64px;
+  left: 570px;
+  top: 261px;
+`;
+
+interface LoginProps {}
+
+export default function Login({}: LoginProps) {
+  const [username, setUsername] = useState<string | undefined>(undefined);
+  const [password, setPassword] = useState<string | undefined>(undefined);
+
+  const onChangeUsername = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setUsername(e.target.value);
+  };
+
+  const onChangePassword = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(e.target.value);
+  };
+
+  const onSubmit = () => {
+    if (!username || !password) {
+      alert("Must input username and password!");
```

Comment:
Does it make sense to both alert the user and show it in the input? I think it's mostly just added fuss to have to click out of the alert

Reply:
I think the alert was just to stop someone from submitting if they haven't filled in both, but when we are actually able to verify if they input a valid username and password then I agree we wouldn't need that alert.